scripts/add_cluster.sh

```
@@ -0,0 +1,154 @@
+while ! kubectl get secret $CLUSTER-vmuser -n observability --context $CTRL_PLANE_CTX &> /dev/null; do
+    sleep 2
+    echo -n "."
+done
+
+echo "vmuser secret created."
+
+# Create vmuser secret
+if ! kubectl get secret vmuser --namespace=observability &> /dev/null; then
+    gum spin --spinner dot --title "Creating vmuser secret..." --show-error -- \
+    kubectl create secret generic vmuser \
+        --from-literal=password=$(kubectl get secret $CLUSTER-vmuser -n observability -o jsonpath="{.data.password}" --context $CTRL_PLANE_CTX | base64 --decode) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)